.teamcity/_self/projects/WebApp.kt

```diff
@@ -367,7 +367,7 @@ object BuildDockerImage : BuildType({
 				fi
 
 				export GH_TOKEN="%matticbot_oauth_token%"
-				. ./bin/add-pr-comment.sh "%teamcity.build.branch%" "calypso-live" <<- EOF
+				. ./bin/add-pr-comment.sh "%teamcity.build.branch%" "calypso-live" <<- EOF || true
```

Comment:
This syntax may seem a bit weird, but that's how Bash defines the heredoc: it treats the doc as a _single word_, replacing the doc start (`<<- EOF`) with the content after the next new line (`Link to live...`) until the document end `EOF`.

So it ends up running `. ./bin/add-pr-comment.sh "%teamcity.build.branch%" "calypso-live" <content> || true`

More info: https://stackoverflow.com/a/7046926